rotkehlchen/db/dbhandler.py

```diff
@@ -1742,7 +1738,7 @@ def edit_exchange(
                 raise InputError(f'Could not update DB user_credentials_mappings due to {e!s}') from e  # noqa: E501
 
         if new_name is not None:
-            exchange_re = re.compile(r'(.*?)_(trades|margins|asset_movements).*')
+            exchange_re = re.compile(r'(.*?)_(trades|margins|history_events).*')
```

Comment:
hmmm this was also not happening before which was a bug right?

Reply:
Yes, I think this must have been a bug. Although not many exchanges were actually using history_events ranges, so it probably didn't affect much.